pandas/tests/extension/base/interface.py

```diff
@@ -53,3 +54,8 @@ def test_is_extension_array_dtype(self, data):
         assert is_extension_array_dtype(data.dtype)
         assert is_extension_array_dtype(pd.Series(data))
         assert isinstance(data.dtype, ExtensionDtype)
+
+    def test_tolist(self, data):
+        result = tolist(data)
```

Comment:
maybe rather test Series[extension].tolist() here? to check it is the same as list(data) (instead of testing this internal method)